src/inverseRobot.js

```diff
@@ -23,8 +23,35 @@
  *
  * @return {object}
  */
+
+function checkUniqueVals(arr) {
+  let duplicate = null;
+
+  arr.forEach((item, indx, self) => {
+    if (self.indexOf(item) !== indx) {
+      duplicate = item;
+
+      return 0;
```

Comment:
`forEach` method doesn't return anything, so it is useless to return values from its callback.